Playbooks/playbook-DBotCreatePhishingClassifier.yml

```diff
@@ -0,0 +1,365 @@
+id: DBotCreatePhishingClassifier
+version: -1
+name: DBotCreatePhishingClassifier
```

Comment:
I think playbook names are usually with spaces in our OOB content, @anara123 WDYT?

Reply:
ok